deep_translator/detection.py

```diff
@@ -2,7 +2,7 @@
 language detection API
 """
 import requests
-from deep_translator.configs import config
+from .configs import config # TODO: Discussion: Could this be moved here and remove configs.py entirely?
```

Comment:
we can remove configs.py

Reply:
removed and config variable set up in detection.py, which is the only module that calls it.